src/component-library/InsightsList/InsightsList.style.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+
+import { theme } from '../theme';
+import { devices } from '../utils/decives';
+
+type InsightsListItemWrapperProps = { hasInfo?: boolean };
+
+export const InsightsListItemWrapper = styled.div<InsightsListItemWrapperProps>`
+  display: flex;
+  flex-direction: column;
+  justify-content: ${(props) => (props.hasInfo ? 'flex-start' : 'space-between')};
+  flex: ${(props) => (props.hasInfo ? '1' : '0 0 auto')};
+  gap: ${(props) => !props.hasInfo && theme.spacing.spacing4};
+
+  &:not(:last-of-type) {
+    padding-bottom: ${theme.spacing.spacing8};
+    border-bottom: ${theme.border.default};
+  }
+
+  @media ${devices.laptop} {
+    &:not(:last-of-type) {
+      padding-bottom: 0;
+      padding-right: ${theme.spacing.spacing8};
+      border-bottom: none;
+      border-right: ${theme.border.default};
+    }
+  }
+`;
+
+export const Card = styled.div`
```

Comment:
Yes, good idea - you can create the card component as part of this PR, and once it's merged remove the duplicated styles. (It can be done straight away, but I'd prefer the changes in a separate PR).